### ISSUE TITLE
[ACR] Separate management plane tests out of data plane pipelines

### ISSUE THREAD

**Issue by annelo-msft** [Azure/azure-sdk-for-net#35256]
Once this is complete, we can remove the blocking-release label from https://github.com/Azure/azure-sdk-for-net/issues/35227

**Comment by annelo-msft** [comment]
Per investigation by @hallipr and @weshaggard, this is too complicated - closing as "won't fix"
